spec/services/capitals_creation_service_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+RSpec.describe CapitalsCreationService do
+  let(:assessment) { create :assessment }
+  let(:assessment_id) { assessment.id }
+  let(:bank_accounts) { attributes_for_list :bank_account, 2 }
+  let(:non_liquid_assets) { attributes_for_list :non_liquid_asset, 2 }
+  let(:request_payload) do
+    {
+      assessment_id: assessment_id,
+      liquid_capital: { bank_accounts: bank_accounts },
+      non_liquid_capital: non_liquid_assets
+    }.to_json
+  end
+
+  subject { described_class.call(request_payload) }
+
+  before { stub_call_to_json_schema }
+
+  describe '.call' do
+    it 'creates bank accounts for this assessment' do
+      expect { subject }.to change { assessment.bank_accounts.count }.by(bank_accounts.count)
+
+      bank_accounts.each do |bank_account|
+        expect(assessment.bank_accounts.find_by!(name: bank_account[:name])[:lowest_balance].to_f).to eq(bank_account[:lowest_balance])
+      end
+    end
+
+    it 'creates bank accounts for this assessment' do
+      expect { subject }.to change { assessment.bank_accounts.count }.by(non_liquid_assets.count)
```

Comment:
it 'creates non_liquid assests for this assessment'

Reply:
fixed